mantidimaging/gui/windows/live_viewer/presenter.py

```diff
@@ -46,20 +45,16 @@ def handle_deleted(self):
         self.view.remove_image()
         self.clear_label()
 
-    def update_image(self, image_path: str):
+    def update_image(self, images_list: list):
         """Update the image in the view."""
-        if not Path(image_path).exists():
-            return
-        if image_path == '':
+        if not images_list and len(images_list) < 1:
```

Comment:
I don't think the `not images_list`. We have a type annotation, so `images_list` must be a list, so `len` should be safe. Or could just check `not images_list`, as that will be false for an empty list.